spec/models/manageiq/providers/embedded_ansible/provider_spec.rb

```diff
@@ -66,4 +66,50 @@
     subject.url = "server.example.com"
     expect(subject.url).to eq("https://server.example.com/api/v1")
   end
+
+  context "DefaultAnsibleObjects concern" do
+    before do
+      subject.save
+    end
+
+    context "with no attributes" do
+      %w(organization credential inventory host).each do |obj_name|
+        it "#default_#{obj_name} returns nil" do
+          expect(subject.public_send("default_#{obj_name}")).to be_nil
+        end
+
+        it "#default_#{obj_name}= creates a new custom attribute" do
+          subject.public_send("default_#{obj_name}=", obj_name.length)
+          expect(subject.default_ansible_objects.find_by(:name => obj_name).value.to_i).to eq(obj_name.length)
+        end
+      end
+    end
+
+    context "with attributes saved" do
+      before do
```

Comment:
Not a fan of before(:context) for the same reason we don't do before(:all) or before(:suite).  You're just begging for test contamination.